test/chia/1.1.0/Chart.yaml

```diff
@@ -14,5 +14,5 @@ sources:
 - https://github.com/orgs/chia-network/packages/container/package/chia
 dependencies:
   - name: common
-    repository: file://../../../library/common/2105.0.0
-    version: 2105.0.0
```

Comment:
why downgrade common version?

Reply:
It's test train, so this change does not affect it. However i've still bumped it. Thanks for pointing it out!